ansible-ipi-install/group_vars/all.yml

```diff
@@ -46,3 +46,19 @@ worker_count: 0
 jumbo_mtu: false
 # set to true only if you requested a public routable VLAN for your cloud in scale lab
 routable_api: false
+
+
+#post-install
+post_install: "{{ lookup('env', 'POST_INSTALL')|default(false, true) }}"
+openshift_toggle_ovn_patch: "{{ lookup('env', 'OVN_PATCH')|default(false, true) }}"
+openshift_ovn_image: "{{ lookup('env', 'OVN_IMAGE')|default('', true) }}"
+kubeconfig_path: "{{ lookup('env', 'KUBECONFIG_PATH')|default('/home/kni/clusterconfigs/auth/kubeconfig', true) }}"
```

Comment:
Can we move this variable to default.yml?

Reply:
done, thanks